pulsar-broker/src/main/java/org/apache/pulsar/broker/service/nonpersistent/NonPersistentSubscription.java

```diff
@@ -431,6 +431,11 @@ public void expireMessages(int messageTTLInSeconds) {
         // No-op
     }
 
+    @Override
+    public void expireMessages(Position position) {
+        // No-op
```

Comment:
Should be throw an error? Or at least log something?

Reply:
will update.